main.py

```diff
@@ -0,0 +1,60 @@
+from monster import Monster
+from hero import Hero
+
+
+def choose_action():
+    return input(
+        "Enter what you wanna do: 1 - attack , 2 - level up, 3 - heal, 4 - defend\n"
+    )
+
+
+def main():
+    hero = Hero("Ari")
+    monster = Monster("HEN", hero.level)
+    attack = hero
+
+    while hero.hp > 0:
+        user_choice = choose_action()
+        hero.increase_coins(1)
```

Comment:
if the increase coins is being done all the time, you could use the same logic for the monster attack, and not write it in every case.          
this should be done at the end of the turn